feat: [PL-26106]: Hierarchial RBAC UI improvements

reetika04 opened the pull request:
##### Summary:

<!--- Add summary of your changes here -->

##### Jira Links:

<!--- Add jira links for your changes here -->

##### Screenshots:
<img width="1045" alt="Screenshot 2022-06-24 at 2 41 51 PM" src="https://user-images.githubusercontent.com/67941036/175808011-5bc1e883-1b50-493f-a8ed-9b5596815961.png">

<!--- Add screenshots for your changes here -->

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

upadhyay-an reviewed (approved):
LGTM!

It'd add more context to code change if you could add before version of screenshot as well.

reetika04 commented:
1. Used ref to scroll to the latest added org.
2. Changed the size of the arrow in collapse
3. Changed the no data text
4. Pre-selection of an org on zero state
